pkg/scaffold/internal/filesystem/mock.go

```diff
@@ -0,0 +1,166 @@
+/*
```

Comment:
Hi @Adirio, 

Is this mock file used in the tests only? If yes, could you please rename the file too mock_test.go? 
Few reasons are; make clear that it is used in the tests only and is not really used in the project implementation, the file not be count as a code that should be coverage by tests by the coveralls.